ARM-template/createUiDefinition.json

```diff
@@ -54,6 +54,49 @@
                             "required": true
                         },
                         "visible": true
+                    },
+                    {
+                        "name": "registryUrl",
+                        "type": "Microsoft.Common.TextBox",
+                        "label": "Registry url",
+                        "defaultValue": "",
+                        "toolTip": "Custom image registry to be used. If you don't understand it, leave it blank.",
```

Comment:
If you want to use a public registry service e.g. Docker Hub, put repository url here. Otherwise, leave it blank.